Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:20.9.0-alpine as builder
+
+# Set directory for the sources to be processed
+WORKDIR /source/build
+
+# Install node depenedencies
+COPY build/package.json package.json
+COPY build/package-lock.json package-lock.json
+RUN npm ci
+
+WORKDIR /source
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+RUN npm ci
+
+# Copy and build the app
+COPY build build
+COPY app app
+COPY tsconfig.json tsconfig.json
+
+RUN npm exec --prefix ./build vendor
+RUN npm run build
+
+# Create kubo container
+FROM ipfs/kubo:latest
+
+WORKDIR /container-init.d
+ADD ipfs.config.sh .
+RUN chmod a+x ipfs.config.sh
```

Comment:
See comment below.  I recommend using the code I have in the Tornado Dockerimage as I don't think what you have here will do what we need.

Reply:
this script/file should not be on the latest changes to this PR